File: two_sum/two_sum.py
def two_sum(targetNumb, ary):
    foundNum = set()
    for num in ary:
        targetValue = targetNumb - num
        if targetValue in foundNum:
            print(targetValue + " + " + num + " = " targetNumb)
            return
        foundNum.add(num)
    return

File: two_sum/two_sum.php
<?php

function twoSum(Int $target_number, Array $array){
    $found_numbers = array();

    foreach($array as $number){
        $target_value = $target_number - $number;
        if(in_array($found_numbers,$target_value)){
            echo("$target_value + $number = $target_number");
            return;
        }
        $found_numbers[] = $target_value;
    }

    return;
}

?>